mkdocs.yml

```diff
@@ -32,17 +32,17 @@ extra:
         - /extra/js/preload_hotjar.js
         - /extra/js/preload.js
     social:
-        - type: github
+        - icon: fontawesome/brands/github
           link: https://github.com/exabyte-io
-        - type: linkedin
+        - icon: fontawesome/brands/linkedin
           link: https://www.linkedin.com/company/mat3ra/
 # Doesn't work with our version. It works from version 7.1.8
 #    analytics:
 #        provider: google
 #        property: UA-69270713-5
 markdown_extensions:
-    - markdown_include.include:
-        base_path: data
+    # - markdown_include.include:
```

Comment:
`markdown_include` is not compatible, check which features are affected.

Reply:
Let's remove it then?